### ISSUE TITLE
Ckpts Permission Error

### ISSUE THREAD

**Issue by scientistdanyal** [DiffPitcher/DiffPitcher#2]
Hi, I updated the ckpts but now i am getting error like this

![Screenshot 2024-03-19 171444](https://github.com/haidog-yaqub/DiffPitcher/assets/127227129/3c3a78e0-8e54-4388-8c83-6d659e21c166)

**Comment by scientistdanyal** [comment]
Hi @haidog-yaqub , will you help me please that why it's give me an error??

**Comment by haidog-yaqub** [comment]
> Hi @haidog-yaqub , will you help me please that why it's give me an error??

Hi, sorry im on vacation last week. could you check it u download all files under this link: https://drive.google.com/drive/folders/1JY98g-gSRq1Xmwt6OtakI9FFrSIaSSug
and put them under the folder of ckpts?

**Comment by scientistdanyal** [comment]
> > Hi @haidog-yaqub , will you help me please that why it's give me an error??
> 
> Hi, sorry im on vacation last week. could you check it u download all files under this link: https://drive.google.com/drive/folders/1JY98g-gSRq1Xmwt6OtakI9FFrSIaSSug and put them under the folder of ckpts?

yes, i did. I am working on it from last 2 months but i did't test it yet.

**Comment by haidog-yaqub** [comment]
> > > Hi @haidog-yaqub , will you help me please that why it's give me an error??
> > 
> > 
> > Hi, sorry im on vacation last week. could you check it u download all files under this link: https://drive.google.com/drive/folders/1JY98g-gSRq1Xmwt6OtakI9FFrSIaSSug and put them under the folder of ckpts?
> 
> yes, i did. I am working on it from last 2 months but i did't test it yet.

Give me one or two days to debug this issue. I updated ckpts for the vocoder last few weeks which not provided in the first version, make sure you also download that one.

**Comment by scientistdanyal** [comment]
Kindly mention your LinkedIn profile or email, sir

**Comment by kazekage92** [comment]
hi @haidog-yaqub , the issue still remain

**Comment by haidog-yaqub** [comment]
> hi @haidog-yaqub , the issue still remain

hi, is it still the permission error? I have tried on different machines but I did not find this issue.
probobly some antivirus software could ban g_05000000, so i update its name to g_05000000.pt

Could you try to use this huggingface link to download code and checkpoints?
https://huggingface.co/Higobeatz/Diff-Pitcher
